gettsim/transfers/kinderzuschl/kinderzuschl.py

```diff
@@ -49,97 +44,89 @@ def kinderzuschl_m_hh(
     -------
 
     """
-    out = _kinderzuschl_nach_vermög_check_m_hh.copy()
-    cond = (~kinderzuschl_vorrang_hh & ~wohngeld_kinderzuschl_vorrang_hh) | (
+    if ((not kinderzuschl_vorrang_hh) & (not wohngeld_kinderzuschl_vorrang_hh)) | (
         anz_rentner_hh > 0
```

Comment:
Is `anz_rentner_hh > 0` in the law or some simplification? If the latter, it might be useful to make a note for that point in time when we'll try to disentangle `grunds_im_alter` and `arbeitsl_geld_2` better.

Reply:
I think the latter.. I have made a note